src/styles/quarks/_quarks.css-properties.scss

```diff
@@ -92,12 +92,22 @@
   --screen-l-max:            74.9375rem; // 1199px (--screen-xl - 1px)
   --screen-xl-max:           89.9375rem; // 1439px (--screen-xxl - 1px)
 
+  /**
+   * FONT FAMILIES
+   * -------------------------------------------------------------------
+   */
+
+  --font-family-base:        'Sun Antwerpen', sans-serif;
+  --font-family-headings:    'Antwerpen Regular', sans-serif;
+  --font-family-monospace:   'Menlo', 'Monaco', 'Consolas', 'Courier New', monospace;
+
   /**
    * BORDERS
    * -------------------------------------------------------------------
    */
 
   --border-radius:           0;
+  --border-width:            .0625rem;
```

Comment:
'thin'?

Reply:
`--border-thin` of `--border-width-thin`? 🙂

Reply:
Ik bedoelde eigenlijk de '.0625rem' vervangen door 'thin', want ik denk niet dat we de borders willen laten meeschalen als de gebruiker een groter lettertype prefereert.

Reply:
Amai, van die property value had ik zelfs nog nooit gehoord. 😅  #TIL

Het probleem is dat bij bepaalde componenten de border niet gezet wordt door `border`, maar door `box-shadow`. Voor die `box-shadow` wou ik dan ook die `--border-width` dan gebruiken: `box-shadow: 0 0 0 var(--border-width) var(--badge-border-color) inset;`. Die `thin` gaat voor `box-shadow` wel niet werken. Dus dan gaan we eens moeten bekijken of we die box-shadow's kunnen omzetten naar een border. 🙂

Reply:
Ahah … geen probleem! In dat geval kunnen we het beter op 1px houden, dan werkt het voor beiden en is eigenlijk even goed dan 'thin' 😊

Reply:
Okidoki, ik zal het dan aanpassen naar `1px`. Dan gaan de borders niet meescalen wanneer de gebruiker zijn UI vergroot. 🙂

Reply:
Yes, klopt!